packages/data/src/world/mm/overworld.yml

```diff
@@ -2270,7 +2270,7 @@
 "Great Bay Coast Fortress":
   region: GREAT_BAY_COAST
   exits:
-    "Great Bay Coast": "underwater_walking"
+    "Great Bay Coast": "underwater_walking || can_dive_small"
```

Comment:
Not sure how you're gonna get past the wooden plank, especially since you can't even dive here anyway due to being entirely underwater.

Reply:
I thought this was referring to leaving pirates fortress. Reverted.